pkgs/development/libraries/libgpod/default.nix

```diff
@@ -27,20 +40,30 @@ stdenv.mkDerivation rec {
     "--with-udev-dir=${placeholder "out"}/lib/udev"
   ] ++ lib.optionals monoSupport [ "--with-mono" ];
 
-  dontStrip = true;
-
-  propagatedBuildInputs = [ glib libxml2 sqlite zlib sg3_utils
```

Comment:
`lib/pkgconfig/libgpod-1.0.pc` contains `Requires: glib-2.0 >= 2.8.0 gobject-2.0 gdk-pixbuf-2.0 libimobiledevice-1.0`

Reply:
There is no gobject right now. Not sure if this should be added.

Reply:
Yeah, gobject and gthread are all part of glib.